db/migrate/20211211170003_create_posts.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePosts < ActiveRecord::Migration[6.1]
+  def change
+    create_table :posts do |t|
+      t.string :string
+      t.text :text
+      t.integer :comments_counter
+      t.integer :likes_counter
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
- [ ] Let us put this into consideration that a post belongs to a user. So, you need to add a column for the `author_id`